framework/db/BaseActiveRecord.php

```diff
@@ -92,7 +92,7 @@
      * @var array|null old attribute values indexed by attribute names.
```

Comment:
I think you should also change the documentation part if you change the data this variable contains